docs/logs/analysis-tab.asciidoc

```diff
@@ -0,0 +1,97 @@
+[role="xpack"]
+[[xpack-logs-analysis]]
+== Detecting and inspecting log anomalies
+
+beta::[]
+
+With the {ml} {anomaly-detect} features enabled,
+you can use the *Log rate* page in the Logs app to automatically detect and inspect log anomalies and
+the log partitions in which they occurred.
+This helps you to spot suspicious behavior without significant human intervention --
```

Comment:
Do we need to clarify this as "potentially suspicious"? As an anomaly doesn't always denote something bad.